### PR TITLE
Some operators for Interactive

### LINKED ISSUE #401: Operators in AngouriMath Interactive go brrrrrr
Override all boolean, equality, comparison, and [built-in math functions](https://fsharp.github.io/fsharp-core-docs/reference/fsharp-core-operators.html) too!

### PR DESCRIPTION
**Pull request by ryyanmapes**
Should close #401 . Does this need complete unit testing?

### CODE REVIEW

File: Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs [hunk 1]
```diff
@@ -12,3 +13,47 @@ let ( / ) a b =
...
+    (sinh (parsed a)).InnerSimplified
+let cosh a = 
+    (cosh (parsed a)).InnerSimplified
+let tanh a = 
+    (tanh (parsed a)).InnerSimplified
+let sqrt a = 
+    (sqrt (parsed a)).InnerSimplified
+let log logbase a = 
+    (log (parsed logbase) (parsed a)).InnerSimplified
+let log10 a =
```
Comment (WhiteBlackGoose) [inline Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs:56]: Why are those in the PR? 🤔

File: Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs [hunk 2]
```diff
@@ -12,3 +13,47 @@ let ( / ) a b =
     ((parsed a) / (parsed b)).InnerSimplified
 let ( ** ) a b =
     ((parsed a).Pow(parsed b)).InnerSimplified
+let ( ~~~ ) a = 
+    (negation (parsed a)).InnerSimplified
+let ( ||| ) a b = 
+    (union (parsed a) (parsed b)).InnerSimplified
+let ( ^^^ ) a b = 
+    (intersect (parsed a) (parsed b)).InnerSimplified
+let ( = ) a b =
```
Comment (WhiteBlackGoose) [inline Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs:22]: Doesn't it override assignment operator? If it doesn't, then it's probably ok

File: Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs [hunk 3]
```diff
@@ -12,3 +13,47 @@ let ( / ) a b =
     ((parsed a) / (parsed b)).InnerSimplified
 let ( ** ) a b =
     ((parsed a).Pow(parsed b)).InnerSimplified
+let ( ~~~ ) a = 
+    (negation (parsed a)).InnerSimplified
+let ( ||| ) a b = 
+    (union (parsed a) (parsed b)).InnerSimplified
+let ( ^^^ ) a b = 
+    (intersect (parsed a) (parsed b)).InnerSimplified
```
Comment (WhiteBlackGoose) [inline Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs:21]: I think... nobody's gonna use those...

**Review by WhiteBlackGoose — COMMENTED** [review]
Also, please, suppress the warnings about < being not recommended to override. We're kinda aware of what we are doing, and this is only available in the Interactive package, which is not shipped for production usage.

File: Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs (hunk 2 above)
Comment (Happypig375) [reply Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs:22]: The assignment operator in F# is `<-`.

File: Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs (hunk 3 above)
Comment (Happypig375) [reply Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs:21]: Intersection should be `&&&`. `^^^` is exclusive or.

Comment (WhiteBlackGoose) [reply Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs:21]: I'm not sure we need a separate operator for it, to be honest. A function is fine imo, this operator would definitely be not as common as >, <, +, -, etc.

File: Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs (hunk 1 above)
Comment (ryyanmapes) [reply Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs:56]: Oh, I wasn't sure if these were included as F# math functions. It looked to me like F# had their own implementations of these, so I overrode them to use the AngouriMath substitutes.

Comment (WhiteBlackGoose) [reply Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs:56]: But we already have them in AngouriMath.FSharp.Functions, don't we?

**Comment by ryyanmapes** [comment]
How should I go about suppressing the warnings? Is there an established suppressions file?

File: Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs (hunk 1 above)
Comment (ryyanmapes) [reply Sources/Wrappers/AngouriMath.Interactive/AggressiveOperators.fs:56]: Oh, right, I guess those would already override them. I'll remove these then.

**Comment by WhiteBlackGoose** [comment]
The IDE you're using should give you an option to supress them. It's very likely that your operators are underlined by it with a line, you know. If you go ctrl+. or alt+enter (depending on the ide) there, it should give you an option to supress them either locally or globally. We want to suppress it for the whole *project*

**Comment by ryyanmapes** [comment]
That button does exist, but it doesn't seem to do anything when pressed on my side? I could just add it to the file manually, but I'm not sure where that is in the project structure.

File: Sources/Wrappers/AngouriMath.Interactive/AngouriMath.Interactive.fsproj [hunk 4]
```diff
@@ -16,7 +16,7 @@
     <PackageLicenseFile>LICENSE.md</PackageLicenseFile>
     <RepositoryType>public</RepositoryType>
     <PackageIcon>icon.png</PackageIcon>
-    <NoWarn>$(NoWarn);NU5100</NoWarn><!-- dll outside of lib/ folder -->
+    <NoWarn>FS0086</NoWarn><!-- dll outside of lib/ folder -->
```
Comment (WhiteBlackGoose) [inline Sources/Wrappers/AngouriMath.Interactive/AngouriMath.Interactive.fsproj:19]: I'd add, not replace
```suggestion
    <NoWarn>$(NoWarn);NU5100;FS0086</NoWarn><!-- dll outside of lib/ folder -->
```

**Pull request merged**